web/src/deprecated-bgio-ui/2d/token.test.js

```diff
@@ -14,6 +14,9 @@ import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
+// avoid error: The tag <g> is unrecognized in this browser.
+jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
I don't think it is a good idea, it will make it hard to debug and we might miss real bugs. Can you undo?

Reply:
Ohhh nevermind , I didn't see it was only in this test